src/ActiveLogin.Authentication.BankId.Api/Models/SignRequest.cs

```diff
@@ -92,8 +92,18 @@ public SignRequest(string endUserIp, string userVisibleData, byte[] userNonVisib
         /// If the personal number is excluded, the client must be started with the AutoStartToken returned in the response.
         /// </param>
         /// <param name="requirement">Requirements on how the auth or sign order must be performed.</param>
-        public SignRequest(string endUserIp, string userVisibleData, byte[] userNonVisibleData, string personalIdentityNumber, Requirement requirement)
+        public SignRequest(string endUserIp, string userVisibleData, byte[]? userNonVisibleData, string? personalIdentityNumber, Requirement? requirement)
```

Comment:
The same here - one more constructor instead of implicit null in the arguments?

Reply:
The recommendation is to only do additions on parameters, not change the order of them:
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/member-overloading?redirectedfrom=MSDN

> X AVOID being inconsistent in the ordering of parameters in overloaded members. Parameters with the same name should appear in the same position in all overloads.

So, we need to have one with all the params where they can be nullable. We could have a few more to catch the secario where you have userNonVisibleData but not personalIdentityNumber, but I think it might be overkill for now? We end up with a ton of overloads :)